lib/flipper/gates/percentage_of_actors.rb

```diff
@@ -30,7 +30,7 @@ def open?(context)
         if Types::Actor.wrappable?(context.thing)
           actor = Types::Actor.wrap(context.thing)
           id = "#{context.feature_name}#{actor.value}"
-          Zlib.crc32(id) % 100 < percentage
+          Zlib.crc32(id) % 100_000 < percentage * 1_000
```

Comment:
I think a comment or breaking out the scaling factor (1000) might be useful here. (this is on the right thing!)